octo/compute-version.sh

```diff
@@ -10,6 +10,11 @@ else
   VERSION=$(git rev-parse --short HEAD)
 fi
 
+MAJOR_VERSION="$(echo "${VERSION}" | awk -F '.' '{print $1 }')"
+MINOR_VERSION="$(echo "${VERSION}" | awk -F '.' '{print $1 "." $2 }')"
+
+echo "::set-output name=version_major::${MAJOR_VERSION}"
+echo "::set-output name=version_minor::${MINOR_VERSION}"
```

Comment:
Minor note, but can you change just the output name to `version-major` and `version-minor`? We're using dashes everywhere else in the code for GHA output names.